docs/docsource/operation/XDM_GUI/bucket_operations/create_a_replication_workflow.rst

```diff
@@ -0,0 +1,45 @@
+.. _Create a Replication Workflow:
+
+Create a Replication Workflow
+=============================
+
+Before creating a replication workflow, you must have a source bucket and a
+destination bucket deployed.
```

Comment:
```suggestion
destination bucket deployed with **versioning enabled**.
```

Reply:
both source and destination must have versioning enabled?

Reply:
Both